plugins/behaviour/compat6/src/Extension/Compat6.php

```diff
@@ -34,8 +37,78 @@ final class Compat6 extends CMSPlugin implements SubscriberInterface
     public static function getSubscribedEvents(): array
     {
         /**
-         * This plugin does not listen to any events.
+         * Note that onAfterInitialise must be the first handlers to run for this
+         * plugin to operate as expected. These handlers load compatibility code which
+         * might be needed by other plugins
          */
-        return [];
+        return [
+            'onAfterInitialiseDocument' => ['onAfterInitialiseDocument', Priority::HIGH],
+        ];
+    }
+
+    /**
+     * Constructor
+     *
+     * @param   DispatcherInterface  $dispatcher  The event dispatcher
+     * @param   array                $config      An optional associative array of configuration settings.
+     *                                            Recognized key values include 'name', 'group', 'params', 'language'
+     *                                            (this list is not meant to be comprehensive).
+     *
+     * @since   5.4.0
+     */
```

Comment:
Shouldn't this be? See further down in this file.
```suggestion
     * @since   6.0.0
     */
```

Reply:
No, see #45371 .